src/ChainOfResponssibility/Poker/Validators/HandValidators/StraightFlushValidation.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Linq;
+
+namespace ChainOfResponssibility.Poker.Validators.HandValidators
+{
+    public class StraightFlushValidation : HandValidation
+    {
+        private StraightValidation straight;
+        private FlushValidation flush;
+
+        public StraightFlushValidation()
+        {
+            straight = new StraightValidation();
+            flush = new FlushValidation();
+        }
+
+        protected override string IsValid(Card[] hand)
+        {
+            if ((!string.IsNullOrEmpty(flush.Validate(hand))) && (!string.IsNullOrEmpty(straight.Validate(hand))))
```

Comment:
While i agree that in general it's possible to do it, it is not usable for this specific program since you have a single chain of validations from RoyalFlush -> HighHand. Nothing in between actually branches out